File: Pattern14.java
// *****
//  ****
//   ***
//    **
//     *

import java.util.Scanner;
public class Pattern14 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the number:");
        int n = in.nextInt();
        for(int i=1;i<=n;i++){
            for(int k=1;k<=i;k++){
                System.out.print(" ");
            }
            for(int j=i;j<=n;j++){
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: PATTERNS/Pattern14/Pattern14.py

# *****
#  ****
#   ***
#    **
#     *

n = int(input("Enter the number:"))

for i in range(1,n+1):
    for k in range(1,i+1):
        print(" ",end='')
    for j in range(i,n+1):
        print("*",end='')
    print()